pkg/component/etcd/etcd/etcd.go

```diff
@@ -214,6 +241,8 @@ func (e *etcd) Deploy(ctx context.Context) error {
 		}
 	}
 
+	maps.Insert(minAllowed, maps.All(e.values.Autoscaling.MinAllowed))
```

Comment:
When `minAllowed` is defined, resource requests could be set to the same values. This avoids rolling the etcd pods on cluster creation.
Otherwise, the pod is created with the default resource request, before VPA kicks in and rolls it.

Reply:
Nice catch 🏈

Reply:
Addressed in 8d0878b8a896cae35c2c73100f08380e9f45d2ad.